docs/ramalama-serve.1.md

```diff
@@ -91,6 +94,42 @@ CONTAINER ID  IMAGE                             COMMAND               CREATED
 7bb35b97a0fe  quay.io/ramalama/ramalama:latest  llama-server --po...  3 minutes ago  Up 3 minutes  0.0.0.0:43869->8080/tcp  MyGraniteServer
 ```
 
+### Generate a kubernetes YAML file named tini
+```
+$ ramalama --nocontainer serve --name tini --generate kube tiny
+# Save the output of this file and use kubectl create -f to import
+# it into Kubernetes.
+#
+# Created with ramalama-0.0.17
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: tini-pod
+  name: tini-pod
+spec:
+  containers:
+  - name: tini
+    image:   quay.io/ramalama/ramalama:latest
+    command: [ "llama-server" ]
+    args:    ['--port', '8080', '-m', '/run/model']
+    ports:
+    - containerPort: 8080
+      hostPort: 8080
+    volumeMounts:
+        - mountPath: /run/model
```

Comment:
This indentation works, but we should keep the way arrays are indented consistent

Reply:
Fixed